nibabel/streamlines/header.py

```diff
@@ -0,0 +1,19 @@
+
+class Field:
```

Comment:
In the previous PR, I had a class that was doing some checks as you said. However, it was too complicated for what it worth and decided to use plain dictionary. Also, there is basically no file format, aside TRK, that has enough useful information (if any) for those checks.

I'm not sure I'm getting what you mean by a tuple of FIELD_NAMES? The goal of having a Field class was to avoid entering the name of the header's field manually, in other words no risk of typos for common header's fields.

That said, I'm pretty this will change as we start supporting more streamlines file format.